tests/cli/test_main.py

```diff
@@ -0,0 +1,143 @@
+"""
+Copyright (c) 2021 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+import json
+import os
+from textwrap import dedent
+
+from flexmock import flexmock
+
+from osbs.cli.main import print_output
+from osbs.tekton import PipelineRun
+
+
+def test_print_output(tmpdir, capsys):
+    """Test print_output function
+
+    Tests:
+      * if STDOUT is correct
+      * if JSON exported metadata are correct
+    """
+    ppln_run = flexmock(PipelineRun(flexmock(), 'test_ppln'))
+    (ppln_run
+     .should_receive('get_info')
+     .and_return(
+        {
+            'metadata': {
+                'annotations': {
+                    # annotations are JSON
+                    'repositories': """{
+                        "primary": ["test1"],
+                        "floating": ["test2a", "test2b"],
+                        "unique": ["test3"]
+                    }""",
+                }
+            }
+        }
+     ))
+    ppln_run.should_receive('has_succeeded').and_return(True)
+    ppln_run.should_receive('status_reason').and_return('complete')
+    (ppln_run
+     .should_receive('get_logs')
+     .and_return([
+        '2021-11-25 23:17:49,886 platform:- - atomic_reactor.inner - INFO - YOLO 1',
+        '2021-11-25 23:17:50,000 platform:- - smth - USER_WARNING - {"message": "user warning"}',
+        '2021-11-25 23:17:59,123 platform:- - atomic_reactor.inner - INFO - YOLO 2',
+     ]))
+
+    export_metadata_file = os.path.join(tmpdir, 'metadata.json')
+    print_output(ppln_run, export_metadata_file=export_metadata_file)
+
+    captured = capsys.readouterr()
+    expected_stdout = dedent("""\
+        Pipeline run created (test_ppln), watching logs (feel free to interrupt)
+        '2021-11-25 23:17:49,886 platform:- - atomic_reactor.inner - INFO - YOLO 1'
+        '2021-11-25 23:17:59,123 platform:- - atomic_reactor.inner - INFO - YOLO 2'
+
+        pipeline run test_ppln is complete
+        primary repositories:
+        \ttest1
+        floating repositories:
+        \ttest2a
+        \ttest2b
+        unique repositories:
+        \ttest3
+        user warnings:
+        \tuser warning
```

Comment:
Maybe user warnings could be more emphasized in the output? They kind of blend in with the rest

Reply:
How?

Reply:
Maybe an empty line, or UPPERCASE as it was before :smile:

Reply:
such inconsistency as UPPERCASE is hurting me, I can add extra line

Reply:
added